packages/react/src/avatar/docs/overview.mdx

```diff
@@ -39,3 +39,21 @@ Use the `action` tone if the Avatar is used within an interactive element such a
 	<Avatar name="William Mead" tone="action" />
 </Flex>
 ```
+
+## Colour
+
+AgDS foreground components respond to the background colour of their parent container. When placed on a dark palette background, the dark palette variant of the component is displayed. When placed on a light palette background, the light palette variant is displayed.
+
+This logic ensures sufficient contrast between foreground and background elements is maintained to meet WCAG 2.1 AA 4.5:1 contrast ratio for text (WCAG 1.4.3) and 3:1 for graphic elements that convey information (WCAG 1.4.11).
+
+```jsx live
+<Fragment>
+	<Flex padding={1.5}>
+		<Avatar name="William Mead" tone="action" />
+	</Flex>
+
+	<Flex background="body" padding={1.5} palette="dark">
+		<Avatar name="William Mead" tone="action" />
+	</Flex>
```

Comment:
I personally would make these `Box` since they don't require any Flex behaiour
```suggestion
	<Box padding={1.5}>
		<Avatar name="William Mead" tone="action" />
	</Box>

	<Box background="body" padding={1.5} palette="dark">
		<Avatar name="William Mead" tone="action" />
	</Box>
```